internal/operators/ocs/validations.go

```diff
@@ -168,7 +168,7 @@ func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID st
 	var err error
 
 	for _, disk := range disks {
-		if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && installationDiskID != disk.ID && disk.InstallationEligibility.Eligible {
+		if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
```

Comment:
We also need to make sure disksize should not be lees than 25G

Reply:
Yes, we are checking that in the next if statement, https://github.com/openshift/assisted-service/pull/2308/files#diff-271f123644f77e7b665d00eab988d90ab0275fde8eedcad9e36ea263592ffefbR172

Reply:
if (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) && installationDiskID != disk.ID && disk.SizeBytes != 0)
when this condition does not meetthen what error we are throwing?

Reply:
We are not throwing an error if this condition is not meeting, we are ignoring the disk as it's either not of the desired type, or its size is zero or it has already been marked for installation of OCP

Reply:
Great!

Reply:
/LGTM